pkgs/development/libraries/libhttpseverywhere/meson.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/meson.build b/meson.build
+index e51d053..5813a77 100644
+--- a/meson.build
++++ b/meson.build
+@@ -75,12 +75,12 @@ if get_option('enable_valadoc')
+ 	valadoc = find_program('valadoc')
+ 	custom_target('apidocs',
+                    input: httpseverywhere_lib_source,
+-                   command: [valadoc, '-o', 'devhelp/httpseverywhere-'+api, '--doclet', 'devhelp', '@INPUT@',
++                   command: [valadoc, '-o', 'devdoc/httpseverywhere-'+api, '--doclet', 'devhelp', '@INPUT@',
+                    '--force', '--pkg', 'gee-0.8', '--pkg', 'json-glib-1.0', '--pkg', 'libarchive', '--pkg', 'libsoup-2.4'],
+-                   output: 'devhelp',
++                   output: 'devdoc',
+                    build_by_default: true)
+-  install_subdir(meson.current_build_dir()+'/devhelp/httpseverywhere-'+api+'/httpseverywhere-'+api,
+-                 install_dir: get_option('datadir')+'/devhelp/books')
++  install_subdir(meson.current_build_dir()+'/devdoc/httpseverywhere-'+api+'/httpseverywhere-'+api,
++                 install_dir: get_option('datadir')+'/devdoc/books')
```

Comment:
I think we just want `install_dir` to be `join_paths(get_variable('devdoc'), get_option('datadir'), 'devhelp', 'books')` or something. Changing the `output` or `command` is not necessary.

You should also add `outputs = [ "out" "devdoc" ];` to the Nix expression for the derivation to be multi-output.

Reply:
`get_variable` does not do, what we want here. There seems to be no possibility to access environment variables in meson, somehow. Any pointers or ideas?

Reply:
Hmm, apparently meson does not offer access to env vars: https://github.com/mesonbuild/meson/issues/9

For now we should probably just add the `outputs` attribute and move `share/devhelp/books` to `$devdoc` in the [setup hook](https://github.com/NixOS/nixpkgs/blob/6bbc3a0b245ccf392a8ddbce41b372c17409f0e9/pkgs/build-support/setup-hooks/multiple-outputs.sh#L137).

Reply:
Okay, all done now, I think!